lib/checkunusedvar.cpp

```diff
@@ -147,7 +147,10 @@ class Variables {
     void clear() {
         mVarUsage.clear();
     }
-    const std::map<nonneg int, VariableUsage> &varUsage() const {
+    const std::map<nonneg int, VariableUsage> &varUsage() const & {
+        return mVarUsage;
+    }
+    std::map<nonneg int, VariableUsage> varUsage() && {
```

Comment:
I do no think we should add/use `&&` unless absolutely necessary.

Especially in this case it makes no sense at all since as this class will only exist once as an global instance.